app/src/main/java/com/sunlightlabs/android/congress/fragments/BillListFragment.java

```diff
@@ -400,18 +401,11 @@ public View getView(int position, View view, ViewGroup parent) {
 			
 			holder.code.setText(Bill.formatCode(bill.bill_type, bill.number));
 
-			if (bill.short_title != null) {
-				String title = Utils.truncate(bill.short_title, 250);
-				holder.title.setTextSize(18);
-				holder.title.setText(title);
-			} else if (bill.official_title != null) {
-				String title = Utils.truncate(bill.official_title, 250);
-				holder.title.setTextSize(14);
-				holder.title.setText(title);
-			} else {
-				holder.title.setTextSize(14);
+            holder.title.setTextSize(14);
+			if (bill.short_title != null)
+				holder.title.setText(Utils.truncate(bill.short_title, 250));
+			else
```

Comment:
The `short_title` is now always populated with the `title`, and always shown in smaller text size (since it could be a long official title).